hash/oric1_cass.xml

```diff
@@ -2439,24 +2631,21 @@ The cassette stops during the game loading
 		</part>
 	</software>
 
-	<software name="missimp">
+	<software name="missimpo">
 		<description>Mission Impossible</description>
 		<year>1984</year>
 		<publisher>No Man's Land</publisher>
+		<notes><![CDATA[
+This game is in two parts: Instructtions + Game.
+On side A: Oric 48K version
+On side B: Atmos version
+]]></notes>
+		<info name="language" value="French" />
+		<info name="region" value="France" />
 		<part name="cass" interface="oric1_cass">
-			<dataarea name="cass" size="23762">
-				<rom name="Mission Impossible (1984)(No Man's Land)(FR).tap" size="23762" crc="c3a97231" sha1="e22d7957cd8c4537d5190ec29aa59df1598f2631"/>
-			</dataarea>
-		</part>
-	</software>
-
-	<software name="missimp1" cloneof="missimp">
-		<description>Mission Impossible (alt)</description>
-		<year>1984</year>
-		<publisher>No Man's Land</publisher>
-		<part name="cass" interface="oric1_cass">
-			<dataarea name="cass" size="39135">
-				<rom name="Mission Impossible (1984)(No Man's Land)(FR)[a no title screen].tap" size="39135" crc="b66e6ab6" sha1="f879ee133a822a3cb23b5eeccb1cd8b7fa37551e"/>
+			<feature name="part_id" value="FACE A - ORIC 48K"/>
+			<dataarea name="cass" size="47175">
+				<rom name="Mission Impossible - Face A.tap" size="47175" crc="1aba6c78" sha1="8b8ac2594946a2fdbfe30520b392a6b335f9ec60"/>
```

Comment:
Is this missing one side?  Should it have a placeholder for the other side?

Reply:
I've added the missing B-side as "nodump" with size=0 (I think that's the way it is done)